File: fix_className.py
import os
import re

directory = r'e:\git\isurupinthu\gitProgects\superMarket\src'

pattern = re.compile(r'<(\w+)\s+className\s*>')

for root, dirs, files in os.walk(directory):
    for file in files:
        if file.endswith('.jsx'):
            file_path = os.path.join(root, file)
            with open(file_path, 'r', encoding='utf-8') as f:
                content = f.read()
            
            new_content = pattern.sub(r'<\1 className="">', content)
            
            if content != new_content:
                print(f"Fixing {file_path}")
                with open(file_path, 'w', encoding='utf-8') as f:
                    f.write(new_content)

File: fix_className.js
const fs = require('fs');
const path = require('path');

const directory = 'e:\\git\\isurupinthu\\gitProgects\\superMarket\\src';

function walk(dir) {
    const files = fs.readdirSync(dir);
    files.forEach(file => {
        const filePath = path.join(dir, file);
        const stats = fs.statSync(filePath);
        if (stats.isDirectory()) {
            walk(filePath);
        } else if (file.endsWith('.jsx')) {
            let content = fs.readFileSync(filePath, 'utf8');
            const newContent = content.replace(/<(\w+)\s+className\s*>/g, '<$1 className="">');
            if (content !== newContent) {
                console.log(`Fixing ${filePath}`);
                fs.writeFileSync(filePath, newContent, 'utf8');
            }
        }
    });
}

walk(directory);
